icu4c/source/test/intltest/intltest.cpp

```diff
@@ -2176,7 +2177,11 @@ UBool IntlTest::assertNotEquals(const char* message,
 // http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertEquals(java.lang.String,%20double,%20double,%20double)
 UBool IntlTest::assertEqualsNear(const char *message, double expected, double actual, double precision) {
     double diff = std::abs(expected - actual);
-    double diffPercent = expected != 0? diff / expected : diff; // If the expected is equals zero, we
+    double diffPercent =
+        expected != 0 ? diff / expected
```

Comment:
This is why I think testing libraries generally use an absolute difference, but I think Hugo has a PR to fix that.

Reply:
I do. Waiting for this PR though, since we're reusing the same function name.